.github/workflows/rust.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   run_checks:
     strategy:
       matrix:
-        rust_toolchain_version: ["1.71", "1.72", "1.73", "1.74"]
+        rust_toolchain_version: ["1.71", "1.72", "1.73", "1.74", "1.75", "1.76", "beta"]
```

Comment:
Are you sure about `beta` though?..

Reply:
+ do we have to support 1.71 still? CI might not like building 75% more versions than it does now.